tests/decodeinputavformat.h

```diff
@@ -44,6 +44,8 @@ class DecodeInputAvFormat : public DecodeInput
     virtual const char * getMimeType();
     virtual bool getNextDecodeUnit(VideoDecodeBuffer &inputBuffer);
     virtual const string& getCodecData();
+    uint16_t getWidth() {return 0;}
```

Comment:
did  you tried use avformat to decode ivf file? Maybe we need return some width & height here too